src/components/attendee/VideoComms/internal/VideoCommsControls.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+  faVideo,
+  faVideoSlash,
+  faVolumeMute,
+  faVolumeUp,
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+interface VideoCommsControlsProps {
+  startAudio?: () => void;
+  stopAudio?: () => void;
+  startVideo?: () => void;
+  stopVideo?: () => void;
+  audioEnabled?: boolean;
+  videoEnabled?: boolean;
+}
+
+export const VideoCommsControls: React.FC<VideoCommsControlsProps> = ({
+  startAudio,
+  stopAudio,
+  startVideo,
+  stopVideo,
+  audioEnabled,
+  videoEnabled,
+}) => {
+  return (
+    <>
```

Comment:
This should probably be an element a `div` or other container with `className="VideoCommsControls"`.

Usually a good idea for each component to have a single element as a container for the rest.

Reply:
The exact HTML structure here will be changing in the new design. I'm happy to leave this as is for now and address this in the redesign. I broadly agree with the idea of having container elements.